packages/tokens/bin/fetch.ts

```diff
@@ -22,33 +23,48 @@ const fetchIconSvgMarkup = async ([iconNodeId, iconUrl]: [string, string]): Prom
 };
 
 const getLibraryData = async (library: LibraryName) => {
+    console.log(`Fetching "${library}" library.`);
     const libraryContent = await fetchLibrary(library);
     writeJsonSync(`./data/${library.toLowerCase()}Library.json`, libraryContent, {spaces: 4});
+    console.log(`${library} library done.`);
```

Comment:
shouldn't it be `console.info` here?

Reply:
Should it? what is the difference?

Reply:
The messages you add seem to be informative messages. So I wonder if it shouldn't be a `console.info`.

In other languages there is often a difference between the display and the prompt, here I don't know if this is the case.

After that it's just a question like that, you don't have to take it into account

Reply:
I did a little research and codewise there doesn't seem to be any difference between the 2. However I think you are right about saying that these are informative message, so I will replace all the console.log by console.info in my next PR